docs/launch_PMM.md

```diff
@@ -0,0 +1,24 @@
+# Launch a Percona Monitoring and Management (PMM) environment
+
+Test-drive PMM from within Percona Platform by generating a free Kubernetes cluster, automatically provisioned with a PMM server, where you can try QAN, DBaaS, Advisor, Alerts and all the features that PMM has to offer.
+
+PMM is an open-source solution for running and managing MySQL and MongoDB clusters on Kubernetes. It provides the observability required to understand database health while offering actionable insights to remediate database incidents or performance issues.  
+
+You can create one free cluster per day. The free clusters have three worker nodes, 4 CPUs/8GB of RAM, and expire in three hours. This makes them ideal for testing out PMM in a native Kubernetes experience.
+
+
+To launch a free PMM environment:
+
+1. In the **Percona Monitoring and Management (PMM)** tab, select **Launch PMM environment**. With that, the cluster is provisioned. Within a few minutes, it’ll be ready to use.
+2. Once the cluster is created, click **Access PMM environment**.
+3. Go to your PMM instance and click <i class="uil uil-database"></i> **DBaaS** on the left menu.
```

Comment:
If you want to test Percona DBaaS feature preview Go to your PMM instance and ...